Add FAQ and product information guide (#584)
iRobotEducation/create3_docs: docs/faq/faq.md

# Create® 3 Educational Robot FAQs

## About the Create® 3 educational robot

### What is the Create® 3 educational robot?
The Create® 3 robot is a programmable robot based on the iRobot® Roomba® i3 vacuum robot.

> Users can program this robot using ROS 2 or Python.

Reviewer: Minor, but I would personally prefer saying
```suggestion
Users can interact with this robot using ROS 2 or Python.
```

There have been three versions of the Create® robot series:

- The original Create® robot, released in 2007. This robot was based on the Roomba® Discovery series robots. It is all-white.
- The Create® 2 robot, released in 2014. This robot is based on the Roomba® 650 robot and includes a green faceplate marked with safe places to drill into the faceplate, once removed from the robot, to mount additional hardware.
- The Create® 3 robot, released in 2022. This robot is based on the Roomba® i3 robot and includes a translucent black faceplate and custom payload bay with holes for mounting.

----

### What age(s) is the Create® 3 educational robot designed for?
The Create® 3 educational robot is a STEM resource for educators, students, and developers.

> It is an advanced platform intended for college and professional learners.

Reviewer: In the FAQ below we also mention "upper high school", should we include it also here?

Author: resolved in upcoming commit by removing "upper high school"

----

### How do I know the Create® 3 educational robot is right for me or my students?
The Create® 3 educational robot is an advanced STEM development platform intended for upper high school students, college students, or lifelong learners.

> The Create® 3 robot works as a node in a ROS 2[^1] graph over Wi-Fi or via its USB-C[^2] host port.

Reviewer: ```suggestion
The Create® 3 robot exposes ROS 2[^1] nodes over Wi-Fi or via its USB-C[^2] host port.
```

Moreover I'm not sure what's the connection between the two sentences here, maybe something is missing ?

----

## Getting Started with the Create® 3

### How do I set up the Create® 3 educational robot?

#### Part 1: Power on and Charge
![An illustration showing the Create® 3 robot being placed on its dock](data/setup-1-1.png "Placing the robot on the dock")
![An illustration showing the Create® 3 charging, with the percentage of its ring glowing white indicating the charge level](data/setup-1-2.png "Charging Indicators")<br>

> To power your robot on, plug your charging dock in and place the robot on the dock.

Reviewer: Should we mention that the dock light will briefly turn on to signal that it got contact with the robot ?

It may take your robot several minutes to charge.
While charging, the Create® 3 light ring will show a spinning, bright white light.
When fully charged, the light ring will glow a dim white.

#### Part 2: Update the Robot
a)	To ensure your robot is up to date, download the latest firmware package available [here](../../releases/overview/).
<br> ![An illustration showing how to press the 1 and 2 buttons](data/setup-2-1.png "Press 1 and 2") <br>
b)	Press and hold Buttons 1 & 2 to activate your robot's hotspot.
When successful, your robot's light ring will show a spinning blue light.
<br> ![An illustration showing how the light ring will change from a cyan swirl to solid cyan once the robot is connected to the computer](data/setup-2-2.png "Connect to robot AP") <br>

> c)	Open your computer's Wi-Fi manager and connect to your robot listed as Create-[xxx].

Reviewer: Should we mention that while connected to this WiFi you won't have Internet access?
